### ISSUE TITLE
Cellzilla 'grow' function

### ISSUE THREAD

**Issue by kazantsevFV** [biomathman/cellerator#4]
Hello!
I am trying to create the model with cell growing. The 'grow' function has the 'reactions' part, where the all set of reactions are set. All of these reactions will be in each cell after generation process.
Question: 
How to add small set of reaction for the particular cell to the model after generation? For example, I want to add some inflow directly into the center of the tissue.

Thank you!

**Comment by biomathman** [comment]
If you don't have cell division, you should be able to go into the reactions and manually edit them after they are generated, but this very complicated thing to do.

I will try my best to answer since my Mathematica has expired some years ago and I no longer work on this software. At the time, nobody ever asked for this feature so it was not coded in, but it would have been a good idea, so we had do it the hard way, which is this: you have to define some fake proteins that act as indicator functions, say I[n] where n is cell number. Make sure I[n]=1 in the cell where you want your reaction to take place and I[n]=0 in the cells where you want your reactions not to occur. Then arrange to have I[n] multiplied times your rate law in such a way that the reaction is effectively unchanged. Although the reaction exists in every cell, it only is effective in the cells where you want them. This is very inefficient, of course, but I don't remember any other way of doing it. It hindsight it seems silly that we did not hard code something like this, or maybe we did, but I just can't remember it. Sorry.

**Comment by kazantsevFV** [comment]
Yes, I would like to do cell divisions on the next step. I have tried to make fake protein to simulate influence on selected cells. But it generates a lot of unnecessary reaction entities for the big tissue.

I would like to have some IF clause criterion for the reaction entity to include it in the concreate cell reactions.
For inner reactions: ifReaction[i]
For Transport reactions: ifReaction[i,j,k]
It should return set of cellzilla reaction objects, that will takes part in generation process, or '0', that means to do nothing with cell.
